### DISCUSSION TITLE
Auto hide side panel

### DISCUSSION THREAD

**Discussion by dncc89** [open-webui/open-webui#4965] [Ideas]
Claude has automatic show/hide side panel when cursor is close to the left edge of the screen, would be great to have something similar.

**Comment by justinh-rahb** [comment]
I find that annoying as heck, no thanks.

**Comment by thiswillbeyourgithub** [comment]
Related to #4426  where I ask how to stop the sidebar from automatically opening.

It's really annoying to me that the sidebar opens up every time I full screen on a vertical screen and every time I put my tablet in landscape mode (if I put it in landscape mode, it's to see more not less!).

I'm using i3wm so it's very frequent for me to trigger the full screen

**Reply by justinh-rahb** [reply]
This sounds more like it would be best solved by making the sidebar hide/show status "sticky".

**Reply by thiswillbeyourgithub** [reply]
I agree. Can this be considered? I checked the code but not knowing svelte nor js/ts I couldn't make a PR

**Comment by lee-b** [comment]
It would be great to auto-hide/auto-unhide when entering/exiting fullscreen mode, respectively. Primarily for immersive fiction/gaming, but also just for focus, I suppose.
